Document transfer memos. (#231)
Concordium/concordium.github.io: source/shared/net/references/transactions.rst

-  ``--amount``: number of GTU tokens to send.
-  ``--receiver``: name or address of the receiver account.

The following flags are for adding a :ref:`transfer memo<glossary-transfer-memo>` to the transfer.

> -  ``--memo``: optional flag for providing a transfer memo as a string. The string will be CBOR encoded and appear on chain as a bytestring representing the string.

Reviewer: ```suggestion
-  ``--memo``: optional flag for providing a transfer memo as a string. The string will be CBOR encoded and included in the memo.
```

> -  ``--memo-json``: optional flag for providing a transfer memo as a JSON file. The JSON will be CBOR encoded and appear on chain as a bytestring representing the provided JSON.

Reviewer: ```suggestion
-  ``--memo-json``: optional flag for providing a transfer memo as a JSON file. The JSON contents of the file will be CBOR encoded and included in the memo.
```

> -  ``--memo-raw``: optional flag for providing a transfer memo as file with raw bytes. The raw bytes will go directly on chain.

Reviewer: ```suggestion
-  ``--memo-raw``: optional flag for providing a transfer memo as is. The contents of the file will be included in the memo as is without any additional encoding.
```
